support/gathering-cluster-data.adoc

```diff
@@ -34,6 +34,10 @@ endif::openshift-origin[]
 // Gathering data about specific features
 include::modules/gathering-data-specific-features.adoc[leveloffset=+2]
 
+== Additional resources
+
+* For information on gathering data about the Custom Metrics Autoscaler, see xref:../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-gather_nodes-pods-autoscaling-custom[Gathering data about your cluster] in the Custom Metrics Autoscaler documentation.
```

Comment:
Since we are supposed to keep add'l ref text to the bare minimum, maybe editing the link name here would be a good compromise?
```suggestion
* xref:../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-gather_nodes-pods-autoscaling-custom[Gathering data about the Custom Metrics Autoscaler in your cluster]
```